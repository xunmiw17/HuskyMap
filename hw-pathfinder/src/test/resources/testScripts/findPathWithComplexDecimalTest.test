CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 2.74638505
AddEdge simpleGraph B C 5.68364563
AddEdge simpleGraph A C 9.72545219

FindPath simpleGraph A C