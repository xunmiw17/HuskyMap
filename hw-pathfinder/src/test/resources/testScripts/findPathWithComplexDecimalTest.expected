created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 2.746 from A to B in simpleGraph
added edge 5.684 from B to C in simpleGraph
added edge 9.725 from A to C in simpleGraph

path from A to C:
A to B with weight 2.746
B to C with weight 5.684
total cost: 8.430